kuma/redirects/redirects.py

```diff
@@ -752,4 +752,30 @@ def locale_redirect(pattern, to, prepend_locale=True, **kwargs):
         r'^?demos',
         '/docs/Web/Demos_of_open_web_technologies',
         permanent=True),
+
+    # Legacy off-site redirects (bug 1362438)
+    # RewriteRule ^contests/ http://www.mozillalabs.com/ [R=302,L]
+    redirect(r'^contests/', 'http://www.mozillalabs.com/', permanent=False),
+
+    # RewriteRule ^es4 http://www.ecma-international.org/memento/TC39.htm [R=302,L]
+    redirect(r'^es4', 'http://www.ecma-international.org/memento/TC39.htm',
+             permanent=False),
+
+
+    # bug 962148 - legacy locales served content instead of redirect
+    # RewriteRule ^en/(.*)$     /mwsgi/en-US/$1 [L,QSA,NE,NC]
+    redirect(r'^en/(?P<path>.*)$', '/en-US/{path}',
```

Comment:
We handle locale redirect already with `LocaleMiddleware`. So why should we implement this redirect here?

Reply:
I think it is cleaner to handle all the legacy redirects in redirects library. I have a long-term goal of completing the migration from MindTouch to Kuma, so I'm 👎 on continuing to maintain the MT locales in their own middleware. But, retiring MindTouch is not an AWS blocker, so removing this commit from the PR.